frontend/cypress.config.js

```diff
@@ -3,6 +3,22 @@ const { defineConfig } = require("cypress");
 module.exports = defineConfig({
     e2e: {
         baseUrl: "http://localhost:3000",
+        // Adding custom task logging, for better a11y output
+        // ref: https://docs.cypress.io/api/commands/task#Usage
+        // https://github.com/component-driven/cypress-axe#using-the-violationcallback-argument
+        setupNodeEvents(on, config) {
```

Comment:
Did you intend to check this in? (seems unrelated to the PR)

Reply:
Yep - it was more related to debugging the e2e failures, etc - in general it adds better logging output for a11y tests